src/Moryx.AbstractionLayer.Resources.Endpoints/ResourceManagementController.cs

```diff
@@ -187,11 +206,11 @@ public ActionResult<ResourceModel> Save(ResourceModel model)
                     resourcesToSave.Skip(1).ForEach(id => _resourceManagement.Modify(id, r => true));
                 });
 
-                return Ok(GetDetails(id));
+                return GetDetails(id);
```

Comment:
Why did you remove the explicit `Ok` here? I like the readability aspect of it.

Reply:
Yeah that is why I added it in a recent commit, problem is GetDetails is already a controller action that returns Ok(resourceModel), i.e. wrapping it in another Ok() call causes the response object to look like this
```json
{
    result: null,
    value: // the actual resource model that should be serialized
}
```
This is not detected by the ui compiler or swagger, but causes errors when saving resources for example.